src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -77,7 +78,12 @@ trait Backend {
   /**
    * Engine functions that don't need a Call context (e.g. read_lines(), read_float(), etc)
    */
-  def engineFunctions: WdlStandardLibraryFunctions
+  def engineFunctions(interface: IOInterface): WdlStandardLibraryFunctions
+
+  /**
+    * Interface to be used primarily by engine functions requiring IO capabilities.
+    */
+  def IOInterface(workflowOptions: WorkflowOptions): IOInterface
```

Comment:
Now that I'm looking at this on a screen larger than an iPhone, I actually disagree w/ @mcovarr although I'd probably call it `ioInterface`